site/main.css

```diff
@@ -85,33 +186,52 @@ hr.footnotes { width: 40%; }
 a.footnote
 {
     text-decoration: none;
+    margin-right: 0.4rem;
 }
 
 ul.toc { list-style: none; }
 
-h5 { font-size: 100%; }
+h5 { font-size: 1.6rem; }
+
+code, kbd, samp
+{
+    color: #1a8a66;
+    color: var(--Code-color, #1a8a66);
+}
 
-code, kbd { color: green; }
 
 /** Typography */
 
 @supports (hyphens: auto) or (-ms-hyphens: auto)
 {
     main
     {
-        text-align: justify;
```

Comment:
https://designforhackers.com/blog/justify-text-html-css/

I tend to agree with this article after many years of trying to use hyphenation. The web still just just not very well optimized for justification because it doesn't do anything "smart" with rivers to aid in the rivers. It's not hard to undelete though :+1:

Reply:
Before the hyphenation support went mainstream, I would agree completely, left align was the lesser of two evils. In the post-hyphens era, not so much. For the most part, it's "almost good" compared to both alternatives (left align and justification without hypens).